src/wrappers/tasks.ts

```diff
@@ -164,6 +167,53 @@ export default class {
     return this.get(createdTask.id);
   }
 
+  public async createFromTemplate(template: ITaskTemplate, orgID: string): Promise<Task> {
+
+    if (template.data[0].type !== TemplateType.Task) {
```

Comment:
I would be inclined to just let the request to the server fail over this check. Also the assertion on the first element is problematic.